packages/forms-web-app/src/dynamic-forms/dynamic-components/task-list/index.njk

```diff
@@ -39,7 +39,7 @@
             <h2 class="govuk-heading-m">
               <span class="app-task-list__section-number">{{loop.index}}. </span> {{section.heading}}
               <strong class="govuk-tag govuk-tag--grey moj-task-list__task-completed">
```

Comment:
I realise this is now merged but I think the styling on this tag needs to change based on the current status. It should use:
`govuk-tag govuk-tag--grey` for "Not started"
`govuk-tag govuk-tag--blue` for "In progress"
and just `govuk-tag` for "Completed"